src/app/pages/BasePage.tsx

```diff
@@ -46,6 +46,9 @@ const styles = {
     flex
     h-full
     w-full
+
+    bg-primaryBg-light dark:bg-primaryBg-dark
+    text-primaryText-light dark:text-primaryText-dark
```

Comment:
Why there are need to specify default text colors?

Reply:
To avoid writing this many times in other parts of the code. And if you need to change the default text color, you can simply override it.